src/server/router.js

```diff
@@ -34,7 +37,9 @@ export const router = {
         taskList,
         taskListIncomplete,
         premisesType,
-        submitSummary
+        submitSummary,
+        receiveMethod
+        // postExitPage - TODO: Uncomment this line to enable the post exit page
```

Comment:
just checking if this is commented out deliberately

Reply:
A/C for this ticket has this page 404ing, if i uncomment this line then it doesnt 404
but its there ready for the ticket to place it in